### PR TITLE
[232] Remove GOV.UK PaaS related code

### PR DESCRIPTION
**Pull request by smneal**
### Context

Remove code and config related to the old GOV.UK PaaS environments. This includes Terraform, GitHub actions and DNS.

### Changes proposed in this pull request

- Remove the paas and modules directories containing old Terraform code
- Remove the old AWS R53 validation records from Azure DNS
- Remove old GitHub workflows and actions
- Remove old Makefile targets and remove unused lines from existing targets that we need to keep e.g. qa, staging for DNS commands

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Inform data insights team due to database changes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,46 +93,22 @@ production_aks:
 .PHONY: qa
```
Comment (divesh987) [inline Makefile:93]: just for my understanding, why are we not also removing these?

Comment (smneal) [reply Makefile:93]: Good question. It's mentioned on the last bullet point in the changes proposed in this pull request, but essentially we still need the DEPLOY_ENV=qa etc for the DNS management.

**Comment by smneal** [comment]
[FindPub - Decommission PaaS environments](https://trello.com/c/6JLhDdSq/232-findpub-decommission-paas-environments)

**Review by saliceti — APPROVED** [review]
LGTM

**Pull request merged**